modules/cats/src/main/tut/README.md

```diff
@@ -39,3 +42,38 @@ then load the config:
 ```tut:book
 loadConfig[MyVecConfig](conf)
 ```
+
+### Using cats type class instances for readers and writers
+
+In order to to put in scope the `cats` type classes for our readers and writers and extend the latter with the extra
+operations provided by `cats`, we need some extra imports:
+
+```tut:silent
+import cats._
+import cats.implicits._
+import pureconfig.module.cats.instances._
+import pureconfig.syntax._
```

Comment:
I'm a bit confused by the imports and maybe you can explain me why we need all four

1. `cats._` is for the typeclasses, no issue here
2. `cats.implicits._` should be `cats.syntax.all`? Do we need the instances for Scala standard library structures?
3. `pureconfig.module.cats.instances._` is the content of this PR and adds the instances for `ConfigReader`, `ConfigWriter` and `ConfigConvert`. No issue here
4. `pureconfig.syntax._` is needed? I see that we are using `Some("abc").toConfig` but that's not required. You can write `someWriter.to(Some("abc"))`, right?